### PR TITLE
gh actions: bump to go 1.20

### PR DESCRIPTION
**Pull request by haircommander**
Signed-off-by: Peter Hunt~ <pehunt@redhat.com>

### CODE REVIEW

File: .github/workflows/integration.yml [hunk 1]
```diff
@@ -12,7 +12,7 @@ jobs:
     steps:
       - uses: actions/setup-go@v2
         with:
-          go-version: '1.18'
+          go-version: '1.19'
```
Comment (saschagrunert) [inline .github/workflows/integration.yml:15]: How about using 1.20 directly?

Comment (haircommander) [reply .github/workflows/integration.yml]: done!

**Review by saschagrunert — APPROVED** [review]
LGTM, CI looks better now but not completely green.

**Comment by haircommander** [comment]
yeah a bit better is a start imo

**Pull request merged**